var/spack/repos/builtin/packages/py-pytaridx/package.py

```diff
@@ -0,0 +1,24 @@
+# Copyright 2013-2022 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack import *
+
+
+class PyPytaridx(PythonPackage):
+    """Python module/library for creating and maintaining a rapidly searchable
+       index for a tar-file. This allows "direct access" of members (files) in
+       the tar archive."""
+
+    homepage = "https://github.com/LLNL/pytaridx"
+    git      = "https://github.com/LLNL/pytaridx.git"
+    url      = "https://github.com/LLNL/pytaridx/archive/refs/tags/v1.0.2.tar.gz"
```

Comment:
```suggestion
    pypi = "pytaridx/pytaridx-1.0.2.tar.gz"
```
See https://spack.readthedocs.io/en/latest/build_systems/pythonpackage.html#pypi-vs-github. You may also need to update the checksum.